public/pages/DefineDetector/components/Settings/Settings.tsx

```diff
@@ -37,10 +37,9 @@ export const Settings = () => {
               <FormattedFormRow
                 fullWidth
                 title="Detector interval"
-                hint="Define how often the detector collects data to generate
-                anomalies. The shorter the interval is, the more real time the 
-                detector results will be, and the more computing resources the 
-                detector will need."
+                hint={[
```

Comment:
Why is this in an array now?

Reply:
otherwise new line will be recognized as actual new line in windows, but not in Ubuntu and macos. Sending you a screenshot offline

Reply:
So not sure, but why is this needed in an array and none of the others? I don't see any explicit newline in the text.

Reply:
the original text is in three lines, 41, 42, 43. The actual Return key will somehow create an actual new line in windows. There's not the only place that long text uses an array to contain the hint text. Not sure why this long text wan't using an array when it was firstly added.

Reply:
I see - I wonder why this only fails for 3 lines vs 2 lines. I see a bunch of places where 2 lines doesn't need an array...

Reply:
Could you point me to where 2 lines doesn't need an array? I believe 2 lines will fail with the exact same reason as well, and I have addressed them all in this PR